pkg/manifests/manifests.go

```diff
@@ -1481,6 +1508,31 @@ func (f *Factory) setupQueryLogFile(p *monv1.Prometheus, queryLogFile string) er
 	return nil
 }
 
+func (f *Factory) setupScrapeProfiles(p *monv1.Prometheus, scrapeProfile ScrapeProfile) error {
+	profiles := make([]string, 0, len(ScrapeProfiles)-1)
```

Comment:
To me it wasn't immediately clear what this function does. Maybe this could be improved by better naming?
E.g. something like `profilesToIgnore` or `ignoredByPrometheus` might make the interaction between the label selector further below a bit clearer.

Reply:
I changed the name and added a method comment, let me know!